tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/TsFileIOWriter.java

```diff
@@ -69,14 +67,19 @@
   }
 
   protected TsFileOutput out;
-  protected List<ChunkGroupMetaData> chunkGroupMetaDataList = new ArrayList<>();
   protected boolean canWrite = true;
   protected int totalChunkNum = 0;
   protected int invalidChunkNum;
   protected File file;
-  private ChunkGroupMetaData currentChunkGroupMetaData;
+  protected List<List<ChunkMetaData>> chunkGroupMetaDataList = new ArrayList<>();
+  protected List<String> deviceList = new ArrayList<>();
+  protected List<ChunkMetaData> chunkMetaDataList = new ArrayList<>();
+  private static Map<Path, List<ChunkMetaData>> timeseriesMetadataMap = new TreeMap<>();
```

Comment:
Why add `static`?  I don't think all `TsFileIOWriter` instances should share this variable, which will lead to serious bugs. Please remove it.

Reply:
Fixed